configure

```diff
@@ -206,8 +206,10 @@ while : ; do
         native_compiler=false;;
     -flambda|--flambda)
         flambda=true;;
+    -with-cplugins|--with-cplugins)
+        with_cplugins=true;;
     -no-cplugins|--no-cplugins)
-        with_cplugins=false;;
+        ;; # Ignored for backward compatibility
```

Comment:
This would break `configure --with-cplugins --no-cplugins`. If anyone actually cares...